orchestration/orchestration.go

```diff
@@ -480,6 +511,20 @@ func (o *Orchestrator) processService(ctx context.Context, input *ServiceOrchest
 		}
 	}
 
+	newTags := []*ecs.Tag{
+		&ecs.Tag{
+			Key:   aws.String("spinup:org"),
+			Value: aws.String(Org),
+		},
+	}
+
+	for _, t := range input.Service.Tags {
+		if aws.StringValue(t.Key) != "spinup:org" && aws.StringValue(t.Key) != "yale:org" {
+			newTags = append(newTags, t)
+		}
+	}
+	input.Service.Tags = newTags
```

Comment:
I know this is not really a big deal in Go, but this code for `newTags` seems to be repeated 3 times. Doesn't it make sense to make it a function?